tests/test_invoices.py

```diff
@@ -0,0 +1,23 @@
+from mollie.api.objects.invoice import Invoice
+
+from .utils import assert_list_object
+
+INVOICE_ID = 'inv_xBEbP9rvAq'
+
+
+def test_list_invoices(client, response):
+    """Retrieve a list of existing invoices."""
+    response.get('https://api.mollie.com/v2/invoices', 'invoices_list')
+
+    invoices = client.invoices.list()
+    assert_list_object(invoices, Invoice)
+
+
+def test_get_invoice(client, response):
+    """Retrieve a single invoice."""
+    response.get('https://api.mollie.com/v2/invoices/%s' % INVOICE_ID, 'invoice_single')
+
+    invoice = client.invoices.get(INVOICE_ID)
+    assert isinstance(invoice, Invoice)
+    assert invoice.id == INVOICE_ID
+    assert invoice.resource == 'invoice'
```

Comment:
When testing the object properties, test all the properties. Coverage is your friend here: https://travis-ci.org/mollie/mollie-api-python/jobs/535306816